pkg/database/postgres/column.go

```diff
@@ -86,7 +86,13 @@ func columnAsInsert(column *schemasv1alpha4.PostgresqlTableColumn) (string, erro
 
 	if postgresColumn.ColumnDefault != nil {
 		value := stripOIDClass(*postgresColumn.ColumnDefault)
-		formatted = fmt.Sprintf("%s default '%s'", formatted, value)
+		if postgresColumn.DataType == "timestamp" {
+			if strings.EqualFold(*postgresColumn.ColumnDefault, "current_timestamp") {
+				formatted = fmt.Sprintf("%s default current_timestamp", formatted)
+			}
+		} else {
+			formatted = fmt.Sprintf("%s default '%s'", formatted, value)
+		}
```

Comment:
The logic here is a little wrong - if the datatype is timestamp but the string is not current_timestamp, formatted is not set.

```suggestion
		if postgresColumn.DataType == "timestamp" && if strings.EqualFold(*postgresColumn.ColumnDefault, "current_timestamp") {
			formatted = fmt.Sprintf("%s default current_timestamp", formatted)
		} else {
			formatted = fmt.Sprintf("%s default '%s'", formatted, value)
		}
```

Reply:
I'm wondering if we should do a more general 'this default is an expression' addition, though - as I imagine that other things like `SERIAL` or `nextval('products_product_no_seq')` would also fail. (examples shamelessly stolen from the [postgresql docs](https://www.postgresql.org/docs/14/ddl-default.html))